src/components/TileSelect.js

```diff
@@ -0,0 +1,42 @@
+import AFRAME from 'aframe';
+import { Vector3 } from 'three';
+
+AFRAME.registerComponent('tile-select', {
+  init: function () {
+    this.tileFloor = document.querySelector('#floor');
+
+    this.closestNode = undefined;
+  },
+
+  tick: function () {
+    // assuming this is a sub object, we want the relative world coordinates
+    let playerPos = new Vector3();
+    this.el.object3D.getWorldPosition(playerPos);
+    let numTiles = this.tileFloor.childNodes.length;
+
+    // 1 because we want to cap distance
+    let minimum = 100;
+    this.closestNode = undefined;
+    let currentPos = new Vector3();
+
+    for (let idx = 0; idx < numTiles; idx++) {
+      //get tile pos
+      //calc distance between player and tile, update min
```

Comment:
We'll have to add a check at some point to make sure that already-picked-up tiles can't be selected with this component (so that for example a tile in a player's hand wouldn't be selected by a different player)

Reply:
My thinking is that once a tile is picked up, we actually delete it and make a new one